<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartidaDialog</class>
 <widget class="QDialog" name="PartidaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Partida</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="partida">
     <property name="title">
      <string>Partida</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="rondaLabel">
        <property name="text">
         <string>Ronda</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="ronda">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grup1">
     <property name="title">
      <string>Equip 1</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="equip1Label">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="equip1"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="taps1Label">
        <property name="text">
         <string>Taps marcats</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="taps1">
        <property name="suffix">
         <string> taps</string>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="bandera1Label">
        <property name="text">
         <string>Bandera pujada</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="bandera1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="extra1Label">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>WhatsThisCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Punts addicionals. Utilitzats per permetre canvis imprevistos a la puntuaci贸. Si s'utilitza, cal especificar una nota de justificaci贸.</string>
        </property>
        <property name="text">
         <string>Punts extra</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="extra1">
        <property name="suffix">
         <string> punts</string>
        </property>
        <property name="minimum">
         <number>-9999</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grup2">
     <property name="title">
      <string>Equip 2</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="equip2Label">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="equip2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="taps2Label">
        <property name="text">
         <string>Taps marcats</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="taps2">
        <property name="suffix">
         <string> taps</string>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="bandera2Label">
        <property name="text">
         <string>Bandera pujada</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="bandera2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="extra2Label">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>WhatsThisCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Punts addicionals. Utilitzats per permetre canvis imprevistos a la puntuaci贸. Si s'utilitza, cal especificar una nota de justificaci贸.</string>
        </property>
        <property name="text">
         <string>Punts extra</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="extra2">
        <property name="suffix">
         <string> punts</string>
        </property>
        <property name="minimum">
         <number>-9999</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="notes">
     <property name="placeholderText">
      <string>Notes (opcional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PartidaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PartidaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
